io_scene_godot/converters/mesh.py

```diff
@@ -126,12 +139,16 @@ def make_arrays(escn_file, export_settings, node, armature_data):
         mesh.calc_normals_split()
         has_tangents = False
 
+    # find the vertex group id of bone weights
+    gid_to_bid_map = find_bone_vertex_groups(node.vertex_groups, armature_data)
```

Comment:
Any reason the `gid_to_bid_map` can't be evaluated inside generate_surfaces to avoid having to pass it in?

Reply:
then the armature_data or part of it need to pass into the `generate_surfaces`,  can avoid passing something into it